### PR TITLE
deployvm: Showing additoinal fields in infocard

### PR DESCRIPTION
**Pull request by davidjumani**
Adds additional fields to the Infocard while creating a vm
- Pod
- Cluster
- Host
- Cpu cores
- Cpu speed
- Memory

Screenshot from 2020-09-03 17-16-10

### CODE REVIEW

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@davidjumani I think we did not include those details to limit the height of the infocard - check on smaller screen if we scroll it well?
@utchoang can you review and test this?

**Comment by davidjumani** [comment]
@rhtyd The user can scroll on the Infocard. The reason I added this was so that the user can see the host & compute resources allocated on the side rather than scrolling up to verify the data passed
Added images when in a smaller screen
Screenshot from 2020-09-07 14-08-49
Screenshot from 2020-09-07 14-08-59

File: src/components/view/InfoCard.vue [hunk 1]
```diff
@@ -280,6 +280,19 @@
             </div>
           </div>
         </div>
+        <div class="resource-detail-item" v-if="resource.networks && resource.networks.length > 0">
+          <div class="resource-detail-item__label">Network(s)</div>
```
Comment (utchoang) [inline src/components/view/InfoCard.vue:284]: @davidjumani can you add a translation for this text?

**Comment by davidjumani** [comment]
@blueorangutan package

**Review by utchoang — APPROVED** [review]
LGTM! @rhtyd.
Thanks @davidjumani

**Pull request merged**